### PR TITLE
Fix/integration feruz

### PR DESCRIPTION
**Pull request by fmuyassarov**
This patch adds credentials to properly checkout airship-dev-tools git repo.

### CODE REVIEW

**Comment by nordixinfra** [comment]
Can one of the admins verify this patch?

File: ci/jobs/capi_bm_integration_tests.pipeline [hunk 1]
```diff
@@ -59,21 +59,33 @@ pipeline {
     }
     stage('Run integration test') {
       options {
-        timeout(time: 60, unit: 'MINUTES')
+        timeout(time: 90, unit: 'MINUTES')
```
Comment (maelk) [inline ci/jobs/capi_bm_integration_tests.pipeline]: let's not increase the timeout here

Comment (fmuyassarov) [reply ci/jobs/capi_bm_integration_tests.pipeline]: Okay

**Pull request merged**